sections/main-collection-banner.liquid

```diff
@@ -13,7 +13,6 @@
   <div class="collection-hero__inner page-width">
     <div class="collection-hero__text-wrapper">
       <h1 class="collection-hero__title">
-        <span class="visually-hidden">{{ 'sections.collection_template.title' | t }}: </span>
```

Comment:
Hello @ChinazoOnwukaike, 

Thanks for looking into Dawn and participating in making a better theme 🙂 

I was looking at the PR and this `<span>` is actually needed. The way it's written seem like it's repeating the collection title twice but it isn't. 

When being called like this: `'sections.collection_template.title' | t`, the theme look for a string in the language files:
```JSON
'sections': {
  'collection_template': {
    'title': 'Collection'
```

So in this case the output when using a screen reader is: 
![](https://screenshot.click/06-39-bl2xs-yrq8l.png)
![](https://screenshot.click/06-32-g0qge-3vonp.png)